### ISSUE TITLE
Improve Auth Page

### ISSUE THREAD

**Issue by Aditya-gupta99** [hieuwu/android-groceries-store#113]
**Is your feature request related to a problem? Please describe.**
The text entered on the sign-in page and sign-up page does not appear in the text view while the text is being typed. As soon as the keyboard is lowered, it can be seen.

**Describe the solution you'd like**
I will fix this by using different text input types and also implementing an outline box style to the text view which makes it looks more user-friendly.

**Comment by hieuwu** [comment]
Good catch. Thank you so much. Please provide the screenshots to compare when you complete the fix
